client/src/api/v2/sensors/data.ts

```diff
@@ -0,0 +1,101 @@
+import { useState, useCallback } from 'react';
+import { useChannel } from '../socket';
+
+/* eslint-disable camelcase */
+export interface SensorData {
+  /** GPS sensor on */
+  gps?: number;
+  /** GPS latitude */
+  gps_lat?: number;
+  /** GPS longitude */
+  gps_long?: number;
+  /** GPS altitude */
+  gps_alt?: number;
+  /** GPS course */
+  gps_course?: number;
+  /** GPS speed */
+  gps_speed?: number;
+  /** Number of GPS satellites */
+  gps_satellites?: number;
+  /** Acceleration X */
+  aX?: number;
+  /** Acceleration Y */
+  aY?: number;
+  /** Acceleration Z */
+  aZ?: number;
+  /** Gyroscope X */
+  gX?: number;
+  /** Gyroscope Y */
+  gY?: number;
+  /** Gyroscope Z */
+  gZ?: number;
+  /** Temperature (Celsius) */
+  thermoC?: number;
+  /** Temperature (Fahrenheit) */
+  thermoF?: number;
+  /** Potentiometer (steering angle) */
+  pot?: number;
+  /** Reed switch velocity */
+  reed_velocity?: number
+  /** Reed switch distance */
+  reed_distance?: number
+  /** Current log file name */
+  filename?: string;
+  /** Time since logging started */
+  time?: number;
+  /** Power */
+  power?: number;
+  /** Cadence */
+  cadence?: number;
+}
+/* eslint-enable camelcase */
```

Comment:
Probably better to update the API specs with consistent casing instead

Reply:
Since it's V2 API I don't think it's work updating. If we leave it as is here then at least it's consistent with the rest of the V2 code.